SimpleSDMPolygons/src/operations/clip.jl

```diff
@@ -0,0 +1,42 @@
+function _get_polygon_from_bbox(bbox)
+    return Polygon(AG.createpolygon([ 
+        (bbox[:left], bbox[:bottom]),
+        (bbox[:left], bbox[:top]),
+        (bbox[:right], bbox[:top]),
+        (bbox[:right], bbox[:bottom]),
+        (bbox[:left], bbox[:bottom]),
+    ]))
+end 
+
+#=
+function clip(
+    poly::Union{Polygon,MultiPolygon},
+    layer::SDMLayer
+)
+
+end
+=#
+
+function clip(
+    feat::Feature,
+    bbox::NamedTuple
```

Comment:
this should probably check that the names in the named tuple are correct

Reply:
fixed